app/components/events/previous-events.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  classNames: ['event-import-section']
```

Comment:
Don't give margin top value now. The semantic header would give some top margin. That _should_ be enough.